src/useProcedureResult/query.js

```diff
@@ -82,10 +82,12 @@ export default function queryProcedureResult({ procedureName, args, forceFetch =
   } = config.getProcedure(procedureName);
 
   return (dispatch, getState, { fetchClient } = {}) => {
-    const cache = getProcedureCache({ procedureName })(getState);
+    const initialCache = getProcedureCache({ procedureName })(getState);
```

Comment:
Only a suggestion, but this could be moved within `if (!forceFetch)` as that's the only scope in which it's utilized.

Reply:
Good suggestion. I have made the change.